src/services/userById.tsx

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/user';
+
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
```

Comment:
If `usersFromServer` is fetched from an API or involves any asynchronous operation, consider making `getUserById` an asynchronous function to handle such cases. This will ensure that user data is correctly retrieved even if there are delays in fetching the data.